pkg/network/node/node.go

```diff
@@ -136,13 +134,11 @@ func New(c *OsdnNodeConfig) (*OsdnNode, error) {
 	case networkutils.NetworkPolicyPluginName:
 		policy = NewNetworkPolicyPlugin()
 		pluginId = 2
-		minOvsVersion = "2.6.0"
```

Comment:
you are now never setting `minOvsVersion` and that code should go away too. Just pull https://github.com/danwinship/sdn/commit/8e3a59b8 in this branch before this commit.